.readme-partials.yaml

```diff
@@ -22,8 +33,9 @@ custom_content: |
 
       // AddressClient#insert()
       System.out.println("\n===============\nAddressClient#insert()\n===============");
-      Operation insertResponse =
-          addressesClient.insert(project, region, Address.newBuilder().setName(address).build());
+      OperationFuture<Operation,Operation> insertResponse =
+          addressesClient.insertAsync(project, region, Address.newBuilder().setName(address).build());
+      Operation insertResponseOperation = insertResponse.get();
       System.out.println(JsonFormat.printer().print(insertResponse) + "\n");
       Thread.sleep(1000L);
```

Comment:
Please revmoe `Thread.sleep()` as it is not needed anymore (.get() on future covers that).